lib/Activity/Provider.php

```diff
@@ -72,13 +72,13 @@ public function parse($language, IEvent $event, IEvent $previousEvent = null) {
 		$event->setIcon($this->urlGenerator->getAbsoluteURL($this->urlGenerator->imagePath('core', 'actions/password.svg')));
 		switch ($event->getSubject()) {
 			case 'webauthn_device_added':
-				$event->setSubject($l->t('You added an WebAuthn hardware token'));
+				$event->setSubject($l->t('You added WebAuthn hardware token'));
```

Comment:
You can have multiple hardware tokens.
So If you added a new one "you added an hardware token".
@Valdnet Does that make sense?

Reply:
Hmm… I looked at the text strings in Transifex and the messages are written differently. Some are with the indefinite article "an" and some are without the indefinite article. Following your line of thought, we can leave this text string unchanged 😉.